client/src/controllers/InitController.test.js

```diff
@@ -148,4 +148,39 @@ describe('InitController', () => {
       ]);
     });
   });
+
+  describe('with a detail', () => {
```

Comment:
Let's use a bit nicer description here.

```suggestion
  describe('when using detail for the dispatched events', () => {
```